code/_onclick/click.dm

```diff
@@ -60,6 +61,12 @@
 		return
 
 	var/list/modifiers = params2list(params)
+	if(isnull(location) && A.plane == CLICKCATCHER_PLANE) //Checks if the intended target is in deep darkness and adjusts A based on params.
+		A = params2turf(modifiers["screen-loc"], get_turf(src.client.eye), src.client)
```

Comment:
```suggestion
		A = params2turf(modifiers["screen-loc"], get_turf(client.eye), client)
```
`src` is implied, it's not necessary. Avoid it unless there's two variables with the same name to require the distinction.